src/typing/flow_js.ml

```diff
@@ -3291,9 +3323,12 @@ let rec __flow cx ((l: Type.t), (u: Type.use_t)) trace =
       rec_flow cx trace (arrt, u)
 
     | (ArrT (_, t, _), (GetPropT _ | SetPropT _ | MethodT _ | LookupT _)) ->
```

Comment:
Why does the `TypeAppT~>MethodT` case above get excluded from `TypeAppExpansion` handling? Is there some edge case there that has an analogue here?